src/pages/settings/AboutPage.js

```diff
@@ -0,0 +1,168 @@
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+
+};
+
+const AboutPage = ({translate, reports}) => {
+    const [reportID, setreportID] = useState(null);
+
+    useEffect(() => {
+        const reportKey = _.findKey(reports, ['lastActorEmail', CONST.EMAIL.CONCIERGE]);
+        if (reportKey) {
+            setreportID(reports[reportKey].reportID);
+        }
+    }, [reports]);
+
+
+    const menuItems = [
+        {
+            translationKey: 'initialSettingsPage.aboutPage.appDownloadLinks',
+            icon: Link,
+            action: () => {
+                Navigation.navigate(ROUTES.SETTINGS_APP_DOWNLOAD_LINKS);
+            },
+        },
+        {
+            translationKey: 'initialSettingsPage.aboutPage.viewTheCode',
+            icon: Eye,
+            iconRight: NewWindow,
+            action: () => {
+                openURLInNewTab(CONST.GITHUB_URL);
+            },
+        },
+        {
+            translationKey: 'initialSettingsPage.aboutPage.viewOpenJobs',
+            icon: MoneyBag,
+            iconRight: NewWindow,
+            action: () => {
+                openURLInNewTab(CONST.UPWORK_URL);
+            },
+        },
+        {
+            translationKey: 'initialSettingsPage.aboutPage.reportABug',
+            icon: Bug,
+            action: () => {
+                if (reportID) { Navigation.navigate(ROUTES.getReportRoute(reportID)); }
+            },
+        },
+
+    ];
+
+
+    return (
+        <ScreenWrapper>
+            <HeaderWithCloseButton
+                title={translate('initialSettingsPage.about')}
+                shouldShowBackButton
+                onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
+                onCloseButtonPress={() => Navigation.dismissModal(true)}
+            />
+            <View style={styles.pageWrapper}>
+                <View style={[styles.settingsPageBody, styles.mb6]}>
+                    <Logo height={100} />
+                    <Text style={[styles.textLabel, styles.alignSelfCenter, styles.mv2, styles.colorMuted]}>
+                        v
+                        {version}
+                    </Text>
+                    <Text style={[styles.textLabel, styles.textP, styles.mv5]}>
+                        Expensify.cash is built by a community of open source developers
+                        from around the world. Come help us build the next generation of
+                        Expensify.
+                    </Text>
+
+                </View>
+            </View>
+            {menuItems.map(item => (
+                <MenuItem
+                    key={item.title}
+                    title={translate(item.translationKey)}
+                    icon={item.icon}
+                    iconRight={item.iconRight}
+                    onPress={() => item.action()}
+                    shouldShowRightArrow
+                />
+            ))}
+            <View style={[styles.sidebarFooter, styles.pAbsolute, styles.b0]}>
```

Comment:
I think this would be better off if we used flexbox instead of absolute positioning. For example, what happens when someone's screen height is short? 

I think we could accomplish this by using something like `alignSelf: flex-end` assuming this child would live within a flexbox parent that has `flex-direction: column` or something like that.

Reply:
Align self was not working that’s why I went with `absolute`. Now I have migrated this `flex` based approach.

<img src="https://user-images.githubusercontent.com/32242596/118303794-7c4ecb00-b503-11eb-922d-0122ad3b1fd0.png" height="400px" />

Reply:
Great! It would be helpful to see this layout on web where the browser height is not very tall. Could you test there and let me know how it looks?

Reply:
The setting page and all `View` based pages are breaking, it would be great if we wrap all of them inside `ScrollView` 
<img width="843" alt="Screenshot 2021-05-14 at 11 10 52 PM" src="https://user-images.githubusercontent.com/32242596/118308531-adca9500-b509-11eb-87c3-800d53a693fc.png">

Reply:
Yeah I agree - ideally that page would scroll if it needs to.

Reply:
Adding ScrollView 👍🏼

Reply:
Done 🚀 

https://user-images.githubusercontent.com/32242596/118310185-e53a4100-b50b-11eb-8011-8026ba4dbb87.mov

Reply:
Nice! Looking pretty good, but I would think the "Read the terms..." should have some margin on top of it so it doesn't get too close to the menu options, and should be closer to the bottom edge of the view:
![image](https://user-images.githubusercontent.com/2319350/118320434-1a657580-b4b1-11eb-887c-3079e7ea112c.png)

Reply:
Dome 🎊 
<img width="902" alt="Screenshot 2021-05-15 at 2 41 45 AM" src="https://user-images.githubusercontent.com/32242596/118332063-1ecc7580-b527-11eb-8d13-3d8912713494.png">

Reply:
Awesome, looks great!